src/foam/comics/v2/DAOUpdateView.js

```diff
@@ -111,9 +111,6 @@ foam.CLASS({
   actions: [
     {
       name: 'save',
-      isEnabled: function(workingData$errors_) {
```

Comment:
This is okay for UCJ's but not for most other types of data. We could either make this configurable or create a sub-class of DAOUpdateView for UCJs. Or find some other solution. But I don't think we should change global behaviour to accommodate UCJs.